src/domain/createBuildRoute/index.js

```diff
@@ -43,4 +42,19 @@ function getParams(args) {
   return undefined;
 }
 
+function normalizeUrl(url) {
+  let result = url.replace(/[^:]\/\//g, '/'); // Gets rid of duplicated slashes (//)
```

Comment:
To fix the first one you might want to do
```js
url.replace(/[^:]\/\//g, (match) => match.replace('//','/'))
```
or
```js
url.replace(/([^:])\/\//g, (match, p1) => `${p1}/`)
```
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/replace#Specifying_a_function_as_a_parameter